languages/python/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,18 @@
+# After
+
+In this lesson, we learned:
+ - Python has a `str` type.
+ - How to find items in a string.
+ - How to manipulate strings to create new strings.
+ - Where to find documentation about string methods.
+
+## More about strings
+
+It's useful to become familiar with Python's string instance methods, formatting, and literal syntax. For a deep dive on what information a string is encoding (or, "how does the computer know how to translate zeroes and ones into letters?"), this blog post is enduringly helpful: https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/
+
+### More about Python logging
+
+Python has a useful built-in logging library, aptly-named `logging`. It handles cases similar
```

Comment:
Maybe link in the text instead of as a separate bit of text?

```suggestion
Python has a useful [built-in logging library](https://docs.python.org/3.8/library/logging.html), aptly-named `logging`. It handles cases similar
```